metas.groovy

```diff
@@ -0,0 +1,137 @@
+//Adapted from module.groovy
+@GrabResolver(name = 'jcenter', root = 'http://jcenter.bintray.com/')
+@Grab(group='org.ajoberstar', module='grgit', version='1.9.3')
+import org.ajoberstar.grgit.Grgit
+import groovy.json.JsonSlurper
+
+Properties properties = new Properties()
+new File("gradle.properties").withInputStream {
+    properties.load(it)
+}
+
+githubHome = properties.alternativeGithubHome ?: "MetaTerasology"

```

Comment:
The key for this differs (see `utility.gradle`) - it is `alternativeGithubMetaHome` this time not `alternativeGithubHome`

Reply:
OK!